docs/proposals/PROPOSAL_E.md

```diff
@@ -166,23 +166,22 @@ When upgrading existing registries, the following manifests MUST be scanned for
 
 Clients SHALL NOT expect manifests uploaded before the [referrers API](#referrers-api) is available on that registry, without using a [digest tag](#digest-tags), will be included in future API responses.
 
-#### Digest Tags
+#### Digest Tag
 
-For registries that do not support the `referrers` API, a tag MUST be pushed for any manifest containing a `refers` descriptor with the following syntax:
+For registries that do not support the `referrers` API, a tag MUST be pushed containing an index of all manifests that refer to the specified manifest, matching the API response above.
+The tag uses the following schema:
 
 ```text
-<alg>-<ref>.<hash>.<type>
+<alg>-<ref>
 ```
 
-- E.g. `registry.example.org/project:sha256-0000000000000000000000000000000000000000000000000000000000000000.0404040404040404.sbom`
+- E.g. `registry.example.org/project:sha256-0000000000000000000000000000000000000000000000000000000000000000`
 - `<alg>`: the digest algorithm
 - `<ref>`: the digest from the `refers` field (limit of 64 characters)
-- `<hash>`: the digest of this artifact (limit of 16 characters)
-- `<type>`: type of artifact for filtering (limit of 5 characters)
```

Comment:
I was avoiding stepping on the existing users of that syntax (see https://github.com/opencontainers/wg-reference-types/pull/61#discussion_r915048316).

Going with that option may reduce the number of conflicts (it still wouldn't help multiple signers with reproducible builds, or multiple SBOM generators of different formats on the same image) but I'd need a full list of every possible type field to avoid running a tag listing to perform a deep copy of an image across registries or repositories.

On the plus side of not including the type in the tag, there's a nice simplicity with both the API response and the tag pull having the same content for the upgraded and existing registries, respectively.